react/Checkbox/Checkbox.less

```diff
@@ -40,16 +40,16 @@
     .focus();
   }
 
-  .input:checked + .label & {
-    background-color: @sk-mid-gray-light;
+  .label & {
+    background-color: @sk-white;
   }
 }
 
 .checkMark {
   display: block;
   width: 100%;
   height: 100%;
-  fill: @sk-black;
+  fill: @sk-focus;
```

Comment:
The point of this change was to make them aligned. So it should be sk-focus in case we do change focus color they are still aligned.